docs/units-and-global-variables.rst

```diff
@@ -262,7 +262,14 @@ Contract Related
     the current contract, explicitly convertible to :ref:`address`
 
 ``selfdestruct(address payable recipient)``:
-    destroy the current contract, sending its funds to the given :ref:`address`
+    Destroy the current contract, sending its funds to the given :ref:`address`
+    and end execution.
+    Note that ``selfdestruct`` has some peculiarities inherited from the EVM.
+    One of them is that the receiving contract's receive function is not executed.
+    Another one is how ``selfdestruct`` relates to ``revert``.
```

Comment:
How's that?

Reply:
selfdestruct is in itself rather complicated. Maybe someone else can gather all the bits from somewhere? Maybe revert is not actually a special case, but selfdestructed contracts are only marked as selfdestructed. For example, if you have a contract that calls itself externally, and the external call uses selfdestruct, then it still returns with a success condition and the contract can continue executing. It might even still have its funds, but it is removed from the state at the end of the transaction, at least if there was no revert in the call chain up.

Reply:
Oh ok. Do you think this example could/should be added here?